backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -217,14 +217,14 @@ public long getPatientsCount(UUID facilityId, boolean isArchived, String namePre
   }
 
   // NO PERMISSION CHECK (make sure the caller has one!) getPatient()
-  public Person getPatientNoPermissionsCheck(UUID id) {
-    return getPatientNoPermissionsCheck(id, _os.getCurrentOrganization());
+  public Person getPatientNoPermissionsCheck(UUID id, boolean showIsDeleted) {
```

Comment:
Rather than updating all the resolvers to manually pass in `false` here, I would recommend overloading this method again so that if `getPatientNoPermissionsCheck(patientId)` is called, it automatically passes in `false` for `showIsDeleted`.